azure-pipelines/verify.signing/no_strongname.txt

```diff
@@ -0,0 +1 @@
+*.dll, strong name signing not needed
```

Comment:
Since you don't want to do any strong name validation, instead of having a whitelist to exclude everything, you can configure the task as：

ExcludeSNVerify: true

See the [doc](https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki?wikiVersion=GBwikiMaster&pagePath=%2FEngineering%20System%20%26%20Tools%2FMicroBuild%2FEnabling%20Signing%20in%20Your%20Projects%2FCodesign%20Verification&pageId=653)

Reply:
done